ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2

```diff
@@ -24,6 +24,24 @@
 | {{ row_contact }} | {{ row_location }} | {{ row_traps_disabled }} | Disabled |
 {%         endif %}
 {%     endif %}
+{%     if snmp_server.engine_ids is arista.avd.defined %}
+
+### SNMP EngineID Configuration
+
+| Type | EngineID (Hex) | IP | Port |
+| ---- | ---------- | -- | ---- |
```

Comment:
!! Not mandatory, just a suggestion. Can you make the ```----``` lines in sync with the above row.